src/Aspire.Dashboard/Otlp/Grpc/OtlpGrpcLogsService.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Dashboard.Authentication;
+using Grpc.Core;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OpenTelemetry.Proto.Collector.Logs.V1;
+
+namespace Aspire.Dashboard.Otlp.Grpc;
+
+[Authorize(Policy = OtlpAuthorization.PolicyName)]
+[SkipStatusCodePages]
+public class OtlpGrpcLogsService : LogsService.LogsServiceBase
+{
+    private readonly ILogger<OtlpGrpcLogsService> _logger;
```

Comment:
`_logger` is unused.

This class is a good candidate for a primary constructor.

Ditto: `OtlpGrpcMetricsService` and `OtlpGrpcTraceService`.